app/src/processing/app/syntax/README.md

```diff
@@ -1,4 +1,14 @@
-# 🐉 Fixing this code: here be dragons. 🐉
+# Replacing our custom version of JEditTextArea
+
+As we have started the migration to Jetpack Compose we will eventually need to replace the JEditTextArea as well.
+
+I think a good current strategy would be to start using `RSyntaxTextArea` for the upcoming p5.js mode as it is a better maintained well rounded library. As noted below, a lot of the current state management of the PDE is interetwined with the JEditTextArea implementation. This will force us to decouple the state management out of the `JEditTextArea` whilst also trying to keep backwards compatibility alive for Tweak Mode and the current implementation of autocomplete.
```

Comment:
Good idea!

To make the sentence clearer, consider specifying that "RSyntaxTextArea is a better maintained, well-rounded library," since it's not immediately obvious whether that description refers to `RSyntaxTextArea` or p5.js.

Also, it might help to clarify that you're proposing to use the upcoming Electron-based p5.js mode as a test bed for integrating `RSyntaxTextArea` into the PDE. This is somewhat implied but it could be clearer.